ReSharper.FSharp/test/src/FSharp.Tests/Daemon/ErrorsHighlightingTest.fs

```diff
@@ -50,3 +50,5 @@ type ErrorsHighlightingTest() =
     [<Test>] member x.``Value not mutable 01``() = x.DoNamedTest()
 
     [<Test>] member x.``Upcast unnecessary 01``() = x.DoNamedTest()
+    
+    [<Test>] member x.``Namespaces cannot contain values or functions``() = x.DoNamedTest()
```

Comment:
Could it be something more concise please?
E.g. `Binding in namespace` or, preferably, the same way the compiler calls it.